client/blocks/reader-full-post/featured-image.jsx

```diff
@@ -4,15 +4,18 @@
 import React from 'react';
 
 export default class FeaturedImage extends React.Component {
```

Comment:
Do we need this component? Looks like the post normalizer is taking care of the missing assets.